[{"type":"message","text":"<https://github.com/vim/vim/commit/04b12697838b232b8b17c553ccc74cf1f1bdb81c|patch 8.2.0695: Vim9: cannot define a function inside a function>\npatch 8.2.0695: Vim9: cannot define a function inside a function\n\nProblem:    Vim9: cannot define a function inside a function.\nSolution:   Initial support for :def inside :def.","ts":"1588628293.052100","attachments":[{"fallback":"GitHub: patch 8.2.0695: Vim9: cannot define a function inside a function · vim/vim@04b1269","id":1,"title":"patch 8.2.0695: Vim9: cannot define a function inside a function · vim/vim@04b1269","title_link":"https://github.com/vim/vim/commit/04b12697838b232b8b17c553ccc74cf1f1bdb81c","text":"Problem: Vim9: cannot define a function inside a function. Solution: Initial support for :def inside :def.","thumb_url":"https://avatars0.githubusercontent.com/u/11618545?s=400&v=4","blocks":null,"service_name":"GitHub","service_icon":"https://a.slack-edge.com/80588/img/unfurl_icons/github.png","thumb_width":48,"thumb_height":48,"from_url":"https://github.com/vim/vim/commit/04b12697838b232b8b17c553ccc74cf1f1bdb81c","original_url":"https://github.com/vim/vim/commit/04b12697838b232b8b17c553ccc74cf1f1bdb81c"}],"edited":{"user":"BPV6T0C1J","ts":"1588628294.000000"},"subtype":"bot_message","bot_id":"BPV6T0C1J","username":"Recent Commits to vim:master","icons":{"image_48":"https://a.slack-edge.com/80588/img/services/rss_48.png"},"replace_original":false,"delete_original":false,"blocks":null}]
